libcontainer/container_linux.go

```diff
@@ -371,6 +371,10 @@ func (c *Container) Signal(s os.Signal) error {
 	// To avoid a PID reuse attack, don't kill non-running container.
 	switch status {
 	case Running, Created, Paused:
+	case Stopped:
```

Comment:
If the pid namespace is private, all processes in this pod ns will be dead after the init process dead.

Reply:
Yes. For the logic, we can simplify it with checking the cgroup only.

Reply:
I don’t think so, with private pid ns and without systemd driver, the cgroup path is always exit even if there is no process, so this will cause an unnecessary function call.